src/components/staking/PageValidator.vue

```diff
@@ -1,12 +1,18 @@
 <template>
   <TmPage
     :managed="true"
-    :data-empty="!validator.operatorAddress"
     :hide-header="true"
     data-title="Validator"
     class="small"
   >
-    <template v-if="validator.operatorAddress" slot="managed-body">
+    <template
```

Comment:
Please use the TmPage logic so this is standardized. Alternatively refactor the logic out of TmPage and create a DataLoading component.
I think we shouldn't duplicate the logic.